README.md

```diff
@@ -36,14 +37,14 @@ You can set the tasks that the crawler runs in `crawler.js`.
 
 ## Testing
 ```sh
-npm test
+gulp test
```

Comment:
yeah and before it was `grunt test`. That's what the npm scripts are for...
The dev just runs `npm test` and doesn't care how those tests actually run